pkg/pillar/cmd/zedagent/parseconfig.go

```diff
@@ -695,7 +699,7 @@ func parseSystemAdapterConfig(config *zconfig.EdgeDevConfig,
 			if err != nil {
 				log.Errorf("parseSystemAdapterConfig: Network with UUID %s not found: %s\n",
 					sysAdapter.NetworkUUID, err)
-				continue
+				return
```

Comment:
Unrelated to this line.. But Can you add forceParse to the log statement at the beginning of the function ?

parseSystemAdapterConfig

log.Infof("parseSystemAdapterConfig: Applying updated config\n"+
		"prevSha: % x\n"+			"prevSha: % x\n"+
		"NewSha : % x\n"+			"NewSha : % x\n"+
		"sysAdapters: %v\n",			"sysAdapters: %v\n",
		systemAdaptersPrevConfigHash, configHash, sysAdapters)

Print forceParse to this statement so that we know why we are parsing the adapters..

Reply:
will do.